.github/workflows/release.yaml

```diff
@@ -71,6 +90,166 @@ jobs:
           files: web5_cli-x86_64-unknown-linux-musl
           repo-token: ${{ secrets.GITHUB_TOKEN }}
 
+  git-tag:
+    runs-on: ubuntu-latest
+    needs: run-ci
+    outputs:
+      RELEASE_TAG: ${{ steps.set-version-and-tag.outputs.RELEASE_TAG }}
+      RELEASE_VERSION: ${{ steps.set-version-and-tag.outputs.RELEASE_VERSION }}
+
+    name: Create Git Tag
+    steps:
+      - id: checkout
+        uses: actions/checkout@v4
+        with:
+          submodules: true
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
it needs to be `TBD_RELEASE_GITHUB_PERSONAL_ACCESS_TOKEN`

because we are actually using our tbd release bot to commit with mvn deploy

Reply:
@ALRubinger I just noticed that we need to add this repo to have permission to this org secret, can you please provide it for us?